tests/Avalonia.Input.UnitTests/TouchDeviceTests.cs

```diff
@@ -72,172 +64,156 @@ public void DoubleTapped_Event_Is_Fired_With_Touch()
 
-            }
+            var pointerPressedExecutedTimes = 0;
+            var tappedExecutedTimes = 0;
+            var isDoubleTapped = false;
+            var doubleTappedExecutedTimes = 0;
+            root.PointerPressed += (a, e) =>
+            {
+                pointerPressedExecutedTimes++;
+                switch (pointerPressedExecutedTimes)
+                {
+                    case <= 2:
+                        Assert.True(e.ClickCount == 1);
+                        break;
+                    case 3:
+                        Assert.True(e.ClickCount == 2);
+                        break;
+                    case 4:
+                        Assert.True(e.ClickCount == 3);
+                        break;
+                    case 5:
+                        Assert.True(e.ClickCount == 4);
+                        break;
+                    case 6:
+                        Assert.True(e.ClickCount == 5);
+                        break;
+                    case 7:
+                        Assert.True(e.ClickCount == 1);
+                        break;
+                    case 8:
+                        Assert.True(e.ClickCount == 1);
+                        break;
+                    case 9:
+                        Assert.True(e.ClickCount == 2);
+                        break;
+                    default:
+                        break;
+                }
+            };
+            root.DoubleTapped += (a, e) =>
+            {
+                isDoubleTapped = true;
+                doubleTappedExecutedTimes++;
+            };
+            root.Tapped += (a, e) =>
+            {
+                tappedExecutedTimes++;
+            };
+            SendXTouchContactsWithIds(InputManager.Instance, touchDevice, root, RawPointerEventType.TouchBegin, 0, 1);
+            SendXTouchContactsWithIds(InputManager.Instance, touchDevice, root, RawPointerEventType.TouchEnd, 0, 1);
+            TapOnce(InputManager.Instance, touchDevice, root, touchPointId: 2);
+            TapOnce(InputManager.Instance, touchDevice, root, touchPointId: 3);
+            TapOnce(InputManager.Instance, touchDevice, root, touchPointId: 4);
+            SendXTouchContactsWithIds(InputManager.Instance, touchDevice, root, RawPointerEventType.TouchBegin, 5, 6, 7);
+            SendXTouchContactsWithIds(InputManager.Instance, touchDevice, root, RawPointerEventType.TouchEnd, 5, 6, 7);
+            TapOnce(InputManager.Instance, touchDevice, root, touchPointId: 8);
+            Assert.Equal(6, tappedExecutedTimes);
+            Assert.Equal(9, pointerPressedExecutedTimes);
+            Assert.True(isDoubleTapped);
+            Assert.Equal(3, doubleTappedExecutedTimes);
+        }
+        private IDisposable UnitTestApp(TimeSpan doubleClickTime = new TimeSpan())
+        {
+            var unitTestApp = UnitTestApplication.Start(
+                new TestServices(inputManager: new InputManager()));
+            var scope = AvaloniaLocator.EnterScope();
```

Comment:
I don't think this is needed here, because `UnitTestApplication.Start` [calls this](https://github.com/AvaloniaUI/Avalonia/blob/master/tests/Avalonia.UnitTests/UnitTestApplication.cs#L38) itself.

Reply:
Should be fixed now.